### DISCUSSION TITLE
Getting token model null on cookie

### DISCUSSION THREAD

**Discussion by shadowoff09** [pocketbase/pocketbase#4980] [Q&A]
Why am getting this %7B%22token%22%3A%22%22%2C%22model%22%3Anull%7D on the cookie (pb_auth) and not the actual cookie for the session?

in the pocketbase dashboard the login is being succeed but then i gives me a invalid token

**Comment by khansu91** [comment]
It is a string that has been JSON.stringify({token: '', model: {}}). Check out pocketbase_auth in localstorage.
